tomls/omnibus-arbitrum-mainnet/perps/global.toml

```diff
@@ -2,7 +2,7 @@
 perps_liquidation_min_keeper_reward_usd = "<%= parseEther('1') %>"
 perps_liquidation_min_keeper_profit_ratio_d18 = "<%= parseEther('0.30') %>"
 perps_liquidation_max_keeper_reward_usd = "<%= parseEther('30') %>"
-perps_liquidation_max_keeper_scaling_ratio_d18 = "<%= parseEther('0.0003') %>"
+perps_liquidation_max_keeper_scaling_ratio_d18 = "<%= parseEther('0.4') %>"
```

Comment:
Want to check on this value. What was the reason to change it?

Reply:
Check out the transactions linked here: https://discord.com/channels/413890591840272394/808181494987620392/1281643453456842795

I think the old setting was incorrect, causing the gas calculations to be low and reducing the liquidation reward.

Reply:
Yep no prob. Need to check these things as they are not directly related to sccp

Reply:
hey yeah, this is purposeful. it should be that much.